### ISSUE TITLE
build warning with `no-bare-strings`: true and ember-i18n installed

### ISSUE THREAD

**Issue by 0xadada** [ember-template-lint/ember-cli-template-lint#406]
I have ember-i18n addon installed, and recieve a warning during the build process:
```bash
The `no-bare-strings` rule must be configured when using a localization framework (`ember-i18n`). To prevent this warning, add the following to your `.template-lintrc.js`:

  rules: {
    'no-bare-strings': true
  }
```

This is my configuration
```js
// .template-lintrc.js
'use strict';

module.exports = {
  extends: ['recommended'],

  rules: {
    // style
    'attribute-indentation': false,/* {
      'element-open-end': 'last-attribute',
      'mustache-open-end': 'last-attribute'
    } */,
    quotes: 'double',

    // enabled rules (in addition to 'recommended')
    'no-bare-strings': true,
    'no-duplicate-attributes': true,
    'no-inline-styles': true
  }
};
```

It appears to be this code causing the warning
https://github.com/ember-template-lint/ember-cli-template-lint/blob/f2f4c19719783ee0a560a2d2da943230fa5b9ff5/broccoli-template-linter.js#L131-L148

If i change 
```js
// change if ('no-bare-strings' in this.linter.config.rules) {
if ('no-bare-strings' in this.linter.config.loadedRules) {
    return;
}
```
it should fix it, but it seems `rules` should be loaded. Instead it is an empty object.

**Comment by 0xadada** [comment]
closed, user-error. I had a syntax error in my config file, and the 0.7.6 version ate these errors. Fixed in a more recent release.
